### DISCUSSION TITLE
-fc parameter filtering responses

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/httpx#932] [Q&A] [answered]
Run: httpx -l in.txt -path dir.txt -sc -fc 200,201,300,301,401,500,501,502,503 -o out.txt

The output includes other status codes not listed in the -fc parameter (e.g. 400).

**Comment by ehsandeep** [comment, marked as answer]
@reconnization looks like you are using the opposite flag. If you are interested in displaying the given status code, you should use, `-mc` instead of `-fc`

**Reply by ghost** [reply]
Oops! You are right, @ehsandeep! Thanks, keep up the great work! :-))
